lib/cli/file-set.js

```diff
@@ -68,8 +68,20 @@ function one(fileSet) {
 function FileSet(cli) {
     var self = this;
 
+    /**
+     * Access files in the file-set.
+     *
+     * @member {Array.<VFile>}
+     */
     self.contents = [];
+
+    /**
+     * Track the file-paths of files when they were added.
+     *
+     * @member {Array.<string>}
+     */
     self.sourcePaths = [];
```

Comment:
Ohhh, nevermind, you were just adding JSDocs. Well, while I appreciate these additions, they are not relevant to the watch PR. Can we move them to a separate commit?